wtp/wtpResultsParser.js

```diff
@@ -95,6 +99,17 @@ const parseTestResponse = (body, rollBarMsg) => {
   return body.data.testId;
 };
 
+const extractLCP = (paintsArray) => {
+  let imagePaints = paintsArray.filter((p) => {
+    return p.event === 'LargestContentfulPaint' && p.type === 'image';
+  });
+  if (imagePaints.length === 0) {
```

Comment:
@yaniv-li I saw cases where there were multiple `LargestImagePaint` events (like in [this](https://www.webpagetest.org/jsonResult.php?test=210613_AiDcKE_9fc00c50ba5bb22a229f191d3ee4217f&pretty=1) case). Not exactly sure why. In that case, perhaps worth taking the largest one ?

Reply:
@mikeys this is why I filter with the type too the other one is text content  if nothing is found (this is what happens if you test a file in github) I go for  "LargestImagePaint" entry

Reply:
I'm saying that there are two `LargestImagePaint` events... one with `time: 441` and another with `time: 976`.

Reply:
added sort by time